wiki/Placeholders.md

```diff
@@ -1470,6 +1470,23 @@ Available gamemodes:
 ```
 ----
 
+- ### **[ClansPro](https://www.spigotmc.org/resources/87515/)**
```

Comment:
Two things:

1. Move this below [Clans-API for Spigot/Clan tag in chat](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#clans-api-for-spigotclan-tag-in-chat)
2. Add this entry to the [List at the top](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#c) (Below the one mentioned in point 1)